src/test/java/bio/terra/janitor/service/janitor/JanitorApiServiceTest.java

```diff
@@ -52,39 +55,104 @@ public void createResource_Duplicates() {
                 ADMIN_USER)
             .getId();
     Map<String, String> retrievedStates =
-        extractStates(janitorService.getResources(resourceUid, ADMIN_USER));
+        extractStates(janitorApiService.getResources(resourceUid, ADMIN_USER));
     assertThat(retrievedStates, Matchers.hasEntry(firstId, "READY"));
     assertThat(retrievedStates, Matchers.aMapWithSize(1));
 
     // Add a resource with the same CloudResourceUid that expired before the first resource.
     String secondId =
-        janitorService
+        janitorApiService
             .createResource(
                 new CreateResourceRequestBody()
                     .resourceUid(resourceUid)
                     .creation(DEFAULT_TIME)
                     .expiration(DEFAULT_TIME.minusMinutes(10)),
                 ADMIN_USER)
             .getId();
-    retrievedStates = extractStates(janitorService.getResources(resourceUid, ADMIN_USER));
+    retrievedStates = extractStates(janitorApiService.getResources(resourceUid, ADMIN_USER));
     assertThat(retrievedStates, Matchers.hasEntry(firstId, "READY"));
     assertThat(retrievedStates, Matchers.hasEntry(secondId, "DUPLICATED"));
     assertThat(retrievedStates, Matchers.aMapWithSize(2));
 
     // Add a resource with the same CloudResourceUid that expired after the first resource.
     String thirdId =
-        janitorService
+        janitorApiService
             .createResource(
                 new CreateResourceRequestBody()
                     .resourceUid(resourceUid)
                     .creation(DEFAULT_TIME)
                     .expiration(DEFAULT_TIME.plusMinutes(20)),
                 ADMIN_USER)
             .getId();
-    retrievedStates = extractStates(janitorService.getResources(resourceUid, ADMIN_USER));
+    retrievedStates = extractStates(janitorApiService.getResources(resourceUid, ADMIN_USER));
     assertThat(retrievedStates, Matchers.hasEntry(firstId, "DUPLICATED"));
     assertThat(retrievedStates, Matchers.hasEntry(secondId, "DUPLICATED"));
     assertThat(retrievedStates, Matchers.hasEntry(thirdId, "READY"));
     assertThat(retrievedStates, Matchers.aMapWithSize(3));
   }
+
+  @Test
+  public void abandonThenBumpResources() throws Exception {
```

Comment:
I think these tests should now move from the JanitorApiServiceTEst to the `TrackedResourceServiceTest`. That's where the logic is.
The JanitorApiServiceTest might not have many if any tets anymore. The only logic in it to be tested is checking that the request is authenticated, which is arguably not worth writing tests for since it's a one liner.

(other tests in this file as well)

Reply:
Done